CONTRIBUTING.md

```diff
@@ -1,34 +1,32 @@
 - Write patches for open bug/feature issues
 
-There are a few guidelines that we need contributors to follow so that we have a
-chance of keeping on top of things.
+To ensure efficiency and organization, we ask all contributors to follow a few guidelines.
 
 ## Getting Started
 
-- Make sure you have a [GitHub account](https://github.com/signup/free).
-- Submit an [issue](https://github.com/jquense/react-big-calendar/issues), assuming one does not already exist.
-  - Clearly describe the issue including steps to reproduce when it is a bug.
-  - Make sure you fill in the earliest version that you know has the issue.
-- Fork the repository on GitHub.
+- **Create a GitHub Account**: Sign up at [GitHub](https://github.com/signup/free) if you don't already have an account.
+- **Report Issues**: Before submitting a new issue, check if it already exists [here](https://github.com/jquense/react-big-calendar/issues).
+  - Describe the issue in detail, including steps to reproduce if it's a bug.
+  - Include the earliest version that you know has the issue.
+- **Fork the Repository**: Make a personal fork of the repository on GitHub to start your work.
 
 ## Making Changes
 
-- Create a topic branch from where you want to base your work.
-  - This is usually the master branch.
-  - To quickly create a topic branch based on master; `git branch master/my_contribution master` then checkout the new branch with `git checkout master/my_contribution`. Better avoid working directly on the
-    `master` branch, to avoid conflicts if you pull in updates from origin.
-- Make commits of logical units.
-  - Write/change `stories` that showcase new features or props
-  - Verify your changes against existing `stories` when necessary
-- Use descriptive commit messages following the [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) format and reference the #issue number.
+- **Create a Topic Branch**: Base your work on the master branch. Create a new branch for your changes with `git branch feat/my_contribution master`, then switch to it with `git checkout feat/my_contribution`. It's best to avoid working directly on the master branch.
+- **Commit Logical Units**:
+  - Add or update `stories` to demonstrate new features or properties.
+  - Ensure your changes are compatible with existing `stories`.
+- **Commit Messages**: Follow the [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) format for your commit messages and include the issue number.
 
 ## Submitting Changes
 
-- Push your changes to a topic branch in your fork of the repository.
-- Submit a pull request to the repository
+- **Push Changes**: Push your changes to your topic branch in your fork.
+- **Create a Pull Request (PR)**: Submit a PR to the main repository. Be sure to:
+  - Write a clear PR title and description detailing what your changes do and why they're needed.
+  - Link your PR to any relevant issues it addresses using phrases like "closes #issue_number".
```

Comment:
Change this to "Link your PR to any relevant issues it addresses by following the [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) format"